src/Contrib.KafkaFlow.Outbox.SqlServer/schema/0002.Table.sql

```diff
@@ -0,0 +1,18 @@
+/* Table */
+IF OBJECT_ID(N'[kafkaflowtest].[dbo].[outbox]', N'U') IS NULL
+BEGIN
+    CREATE TABLE [kafkaflowtest].[dbo].[outbox](
```

Comment:
We shouldn't specify the DB here. And if we change to `outbox` as the schema (to keep in line with the PG implementation) that would change to `[outbox].[outbox]`

e.g:
```SQL
IF OBJECT_ID(N'[outbox].[outbox]', N'U') IS NULL
BEGIN
    CREATE TABLE [outbox].[outbox](
```